wombat-admin/src/app/page.tsx

```diff
@@ -41,12 +41,63 @@ export default function Home() {
   };
 
 
+  const [requests, setRequests] = useState([]);
+
+  useEffect(() => {
```

Comment:
So its generally a good practice to wrap useEffect() into a custom hook, rather than having it directly in the component. This is useful in separating out the useEffect code, as well as making sure that the effect isn't over applied if the page is affected / refreshed.

Would it be possible to extract both the access_request and users `useEffect()` called into custom hooks. Maybe take in the ID value of the access_request or user to retrieve (optionally) so that it could be added to the URL if present. That way, anywhere we need to get these details we can just use these hooks. Guidance on how to create them is here:

https://react.dev/reference/react/useEffect#wrapping-effects-in-custom-hooks

Reply:
Also, with custom hooks we could move the fetch code closer to where it is needed. Rather than having it loaded on the main page we could maybe avoid passing the prop and have the dashboard or daily_visit pages contain it.

Reply:
fixed. once the PR is merged will work on fetching only the manual override requests